website/pages/docs/platform/k8s/helm/examples/standalone-tls.mdx

```diff
@@ -135,9 +88,8 @@ TMPDIR=/tmp
           - digital signature
           - key encipherment
           - server auth
-        EOF
-        ```
-        -> `CSR_NAME` can be any name you want. It's the name of the CSR as seen by Kubernetes
+       ```
```

Comment:
```suggestion
        EOF
        ```
```

Reply:
:+1:

Reply:
Re-copied from existing examples to resolve rebasing issues.